spring-rabbit/src/main/java/org/springframework/amqp/rabbit/core/RabbitTemplate.java

```diff
@@ -245,7 +245,7 @@
 
 	private boolean userCorrelationId;
 
-	private boolean usePublisherConnection;
+	private boolean usePublisherConnection = true;
```

Comment:
We need to change the javadoc instead. It turned out that `true` is not a safe default and will break many existing applications. 

The problem is that any auto-delete, exclusive `Queue` beans in the application context will be declared on the producer connection and any listener containers that consume from those queues will fail to start because they can only be accessed on the connection on which they were declared.